src/Microsoft.DotNet.Interactive.Formatting.Tests/PlainTextFormatterTests.cs

```diff
@@ -475,26 +509,11 @@ public void ReadOnlyMemory_of_char_is_formatted_like_a_string()
                 writer.ToString().Should().Be("Hi!");
             }
 
-            [Fact]
```

Comment:
These tests are removed because the returned formatter is now often a composition of possible formatters (using AnonymousTypeFormatter) and its not useful to check the runtime type of the formatter on return.

To be honest I think we should remove `HtmlFormatter<T>` and `PlainTextFormatter<T>`, ITypeFormatter<T> etc. as actual public implementation types of ITypeFormatter, just use them as part of the API, and have all formatters be either delegates or ITypeFormatter (non-generic). THis will greatly reduce the API surface area.

Reply:
I think this simplification makes sense. I wonder whether `HtmlFormatter<T>` and `PlainTextFormatter<T>` have useful, distinct APIs that people will want to use, but most likely if people just implement their own `ITypeFormatter` the capabilities become much more open-ended anyway.